File: oms/src/main/java/com/bnr/oms/messaging/dto/QueueMessage.java
package com.bnr.oms.messaging.dto;


import java.util.Date;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Builder
public class QueueMessage {

  private String orderId;

  private Date deliveryTime;

  private String action;

  private Integer quantity;

  private String callback;

}

File: notification/src/main/kotlin/com/bnr/notification/dto/QueueMessage.kt
package com.bnr.notification.dto

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_NULL)
data class QueueMessage(
        var orderId: String,
        var deliveryTime: String?,
        var action: String,
        var quantity: Int?,
        var callback: String?
)
